gorm_logger.go

```diff
@@ -21,11 +21,10 @@ func (this GormLogger) Print(values ...interface{}) {
 		source := fmt.Sprintf("%v", values[1])
 		lastIndex := strings.LastIndex(source, "/") + 1
 		writeGormLog(source[lastIndex:])
-
 		if level == "sql" {
 			// duration
 
-			messages := fmt.Sprintf("%s[ %.2fms] ", COLOR_OFF, float64(values[2].(time.Duration).Nanoseconds()/1e4)/100.0)
+			messages := fmt.Sprintf("%s[ %.2fms] ", COLOR_PURPLE, float64(values[2].(time.Duration).Nanoseconds()/1e4)/100.0)
```

Comment:
@kaizer666 Colors are corresponding with log level.
You can see the code at here https://github.com/canhlinh/log4go/blob/master/log4go.go#L63

In this case, I was using COLOR_OFF to disable color on `GORM`. I don't remember why.

If you want to change the color to `COLOR_PURPLE`, please send me your screen shoot. I'll take a look